client/session.go

```diff
@@ -1224,79 +1244,104 @@ func (s *session) FetchBlocksMetadataFromPeers(
 	var (
 		metadataCh = make(chan blocksMetadata, blocksMetadataChannelInitialCapacity)
 		errCh      = make(chan error, 1)
-		m          = s.streamFromPeersMetricsForShard(shard, resultTypeMetadata)
+		m          = s.newPeerMetadataStreamingProgressMetrics(shard, resultTypeMetadata)
 	)
 
 	go func() {
-		errCh <- s.streamBlocksMetadataFromPeers(namespace, shard, peers,
-			start, end, metadataCh, m)
+		errCh <- s.streamBlocksMetadataFromPeers(
+			namespace, shard, peers, start, end, metadataCh, m, FetchBlocksMetadataV1)
```

Comment:
Is it cool to leave this on the old endpoint only? Presumably if it works for the bootstrap it will also work for this